dnf-docker-test/README.md

```diff
@@ -0,0 +1,7 @@
+Generate documentation
```

Comment:
I want to split project documentation (which is outdated) and specific things which are useful only to some people.